content/releases/q/release-3.22.0.md

```diff
@@ -0,0 +1,12 @@
+---
+url: "/releases/q-3.22.0/"
+date: 2025-04-30
+type: release-note
+version: 3.2.0
```

Comment:
```suggestion
version: 3.22.0
```

Reply:
Good spot! Fixed in [f1e1f9a](https://github.com/apache/camel-website/commit/f1e1f9a13db039b1dd5f42a46630eac0854d5f9d).